libevmasm/Assembly.cpp

```diff
@@ -83,7 +83,7 @@ AssemblyItem const& Assembly::append(AssemblyItem const& _i)
 {
 	assertThrow(m_deposit >= 0, AssemblyException, "Stack underflow.");
 	m_deposit += _i.deposit();
-	m_items.push_back(_i);
+	m_items.emplace_back(_i);
```

Comment:
I wouldn't do it here. Since ``_i`` is a constant reference, this will have to copy anyways, even with ``emplace_back``. And ``emplace_back`` is less restrictive: it will call explicit constructors, for which ``push_back`` would generate an error, so: no gain from ``emplace_back``, but less secure due to more unnoticed conversions.

Reply:
Or you change the function signature and pass by value instead of const reference and then move here... then the caller could decide whether he passes the argument as copy or by moving. But I guess it's not important enough to overthink it.

Reply:
Yeah, you might notice that my later replacements were smarter, I basically learned ;)